lib/oli/rendering/content/html.ex

```diff
@@ -179,7 +179,7 @@ defmodule Oli.Rendering.Content.Html do
   def tc(%Context{} = _context, next, attrs) do
     [click_class, audio_element, play_code, _] =
       case attrs["audioSrc"] do
-        nil -> ["", "", ""]
+        nil -> ["", "", "", ""]
```

Comment:
Oops, the audio_player added a return value a while back, guess I missed this one.